scripts/NewsDown/main.py

```diff
@@ -1,9 +1,10 @@
 import requests
-from bs4 import BeautifulSoup
+import datetime
 import img2pdf
-from tkinter import ttk, Tk, LabelFrame, Frame, filedialog
+from bs4 import BeautifulSoup
+from tkinter import ttk, filedialog
+from tkinter import *
```

Comment:
Why are you importing everything from tkinter?

Reply:
By importing all the functions and methods, we can use the inbuilt functions or methods in a particular application without importing them implicitly.
you can see this [Why do we use import * and then ttk in TKinter?](https://www.tutorialspoint.com/why-do-we-use-import-and-then-ttk-in-tkinter)

Reply:
I think the previous format was fine, only importing the necessary modules and functions instead of importing everything in bulk.